docs/extras/nandrestore.md

```diff
@@ -1,36 +1,39 @@
-## Restore a nand backup to the switch
+## Restore a NAND backup to the switch
 
-!!! danger "Warning:"	
-	-This will reset all of your saves, games, system version and other system settings back to a point when you made the the nand backup. Keep this in mind, as you probably don't have to restore a nand backup unless you have bricked your switch or want to go back online safely after using cfw.
+!!! Danger "Warning:" 	
+	-This will reset your system version, all of your saves, games, and other system settings back to a point from when you made the the NAND backup. Keep this in mind, as you probably don't have to restore a NAND backup unless you have bricked your switch or want to go back online safely after using cfw.
 	
-	-If you're going to restore an old nand which will downgrade your firmware it's best to create a second NAND backup before restoring the first one in case something goes wrong.
+	-If you're going to restore an old NAND which will downgrade your firmware it's best to create a second NAND backup before restoring the first one in case something goes wrong.
 
 ### What you need:
-- Your `Rawnand.bin` (Combined or in 15 or 30 parts)
+- Your `RawNAND.bin` (Combined or in 15 or 30 parts)
 - Your `BOOT0` and `BOOT1`
-	- If you're missing any of the files above, ask for further help in the [Nintendo Homebrew](https://discord.gg/C29hYvh) Discord server.
-- The [Hekate](https://github.com/CTCaer/hekate/releases/) Payload
+	- If you're missing any of the files above, ask for further help in the [Nintendo Homebrew](https://discord.gg/C29hYvh) Discord server
+- The <a href="https://github.com/CTCaer/hekate/releases/" target="_blank">Hekate</a> Payload
 - A microSD card larger than 32gb
 
 ### Instructions:
 
-Before we start, check if you have a tree of folders called `backup/[8 Character NAND id]/restore` on your sd card
+Before we start, check if you have a tree of folders called `backup/[8 Character NAND id]/restore` on your sd card.
 
-!!! warning "If you don't see a backup or [8 Character NAND id] folder on your sd card:"
-	1. Launch the latest hekate payload on your switch.
-	2. Navigate to `Tools > Backup > Backup eMMC BOOT0/1`, and press the power button to select the option.
-	3. Wait for the process to complete. When it's completed, you now have a `backup/[8 Character NAND id]/restore` folder on your sd card. Continue with step 1 of the guide.
+!!! Warning "If you don't see a backup or [8 Character NAND id] folder on your sd card:"
```

Comment:
Inconsistent, this should be `warning`, not `Warning`.

Reply:
Fixed to be consistent with the others